packages/ocaml/ocaml.5.1.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+synopsis: "The OCaml compiler (virtual package)"
+description: """
+This package requires a matching implementation of OCaml,
+and polls it to initialise specific variables like `ocaml:native-dynlink`"""
+maintainer: "platform@lists.ocaml.org"
+depends: [
+  "ocaml-config" {>= "2"}
```

Comment:
```suggestion
  "ocaml-config" {>= "3"}
```

and the ocaml.5.0.0 package needs this update as well

Reply:
Updated.